tests/phpunit/ExtractorTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\FtpExtractor\Tests;
+
+use Keboola\FtpExtractor\ConfigDefinition;
+use Keboola\FtpExtractor\FtpExtractorComponent;
+use Keboola\Temp\Temp;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\Filesystem\Filesystem;
+
+class ExtractorTest extends TestCase
+{
+    public function testConnection(): void
+    {
+        $temp = new Temp('ex-storage');
```

Comment:
typo

Reply:
mělo by být ex-ftp, ale není to něijak kritické

Reply:
je to nahodny nazev a jelikoz je nepovinny, tak si myslim, ze tam nemusi byt vubec :)